Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_From_Path_-_Generic_V2.yml

```diff
@@ -0,0 +1,306 @@
+
+        Input:
+        * Credentials - credentials to use when trying to deploy Demisto Dissolvable Agent (D2)  (default: Admin)
+        * ${Endpoint.Hostname} - deploy agent on target endpoint
+        * ${File.Path} - file's path to collect
+      playbookName: Get File Sample From Path - D2
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      Credentials:
+        simple: Admin
+      Hostname:
+        simple: ${Endpoint.Hostname}
+      Path:
+        simple: ${File.Path}
+    separatecontext: false
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 1,
+          "y": 355
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+  "11":
+    id: "11"
+    taskid: 906138a1-e3ee-4711-889a-4ebcc8c2a1c6
+    type: title
+    task:
+      id: 906138a1-e3ee-4711-889a-4ebcc8c2a1c6
+      version: -1
+      name: Use VMware Carbon Black EDR (Live Response API)
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "12"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "12":
+    id: "12"
+    taskid: 931591d2-1338-45bf-8083-0a91db928068
+    type: playbook
+    task:
+      id: 931591d2-1338-45bf-8083-0a91db928068
+      version: -1
+      name: Get File Sample From Path - VMware Carbon Black EDR - Live Response API
+      playbookName: Get File Sample From Path - VMware Carbon Black EDR - Live Response
+        API
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      Path:
+        complex:
+          root: inputs.Path
+      Sensor_ID:
+        complex:
+          root: inputs.Sensor_ID
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 510
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 705,
+        "width": 859,
+        "x": 1,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: UseD2
+  value: {}
+  required: false
+  description: |-
+    Determines whether a D2 agent will be used to retrieve the file.
+    Default is no.
+  playbookInputQuery:
+- key: Hostname
+  value: {}
+  required: false
+  description: Hostname of the machine on which the file is located.
```

Comment:
IP? same as before

Reply:
Not relevant, there are no option to retrieve file from any integration in those playbook by IP.